frontend/packages/console-shared/src/components/spotlight/Spotlight.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import cx from 'classnames';
+import Portal from '@console/shared/src/components/popper/Portal';
+import './Spotlight.scss';
+
+type SpotlightProps = {
+  selector: string;
+  animate?: boolean;
+  hasBackdrop?: boolean;
```

Comment:
There's a bug where if `hasBackdrop === false && animate === false` then this happens:
![image](https://user-images.githubusercontent.com/14068621/100130674-02416180-2e51-11eb-8efd-be362a044e72.png)